articles/how-artificially-intelligent-chatbots-have-impacted-healthcare-industry/index.md

```diff
@@ -0,0 +1,59 @@
+### Introduction
+In recent times, organizations have adopted the benefits offered by AI-powered [virtual assistants](https://searchcustomerexperience.techtarget.com/definition/virtual-assistant-AI-assistant) to automate routine tasks and give their clients attention in customer services, payments, and marketing.
+
+The healthcare industry has now adopted this trend. [AI](https://www.ibm.com/cloud/learn/what-is-artificial-intelligence) has created an environment where the patient outcome has been improved with minimum costs involved.
+
+AI technology is not new to healthcare. The urge for cost reduction and improve patient outcomes and data availability have led to various [innovations](https://innolytics-innovation.com/what-is-innovation/) in the industry, ranging from early disease diagnosis to the introduction of life-saving medicines.
+
+With AI, powerful new opportunities are created across the industry. Furthermore, the latest development is equally promising.
+
+### How do chatbots work?
+[Chatbots](https://www.drift.com/learn/chatbot/) utilizes Natural Language Processing ([NLP](https://machinelearningmastery.com/natural-language-processing/)) to understand, [parse](https://techterms.com/definition/parse) text and speech effectively. Through a user-friendly interface and mimic human conversation either through a standalone or web application.
```

Comment:
Chatbots utilize NLP to understand, parse and process text and speech. 

The second line is not clear. It feels discontiguous to the previous sentence. Modify it.